doc/sphinx/Pacemaker_Explained/acls.rst

```diff
@@ -21,9 +21,11 @@ ACL Prerequisites
    
 In order to use ACLs:
 
-* The Pacemaker software must have been built with ACL support. If the output
-  of the command ``pacemakerd --features`` contains ``acls``, your installation
-  supports ACLs.
+* Prior to the 2.1.0 release, the Pacemaker software had to have been built
+  with ACL support. If you are using version 2.1.0 or older, your
```

Comment:
s/version 2.1.0 or older/an older release/ ... 2.1.0 itself is fine. Let's move this to the bottom of the list as well, and move the enable-acl item to the top, since as time goes on this will be less and less relevant